code/datums/uplink/uplinks.dm

```diff
@@ -1,42 +1,44 @@
-//This could either be split into the proper DM files or placed somewhere else all together, but it'll do for now -Nodrak
-
-/*
-
-A list of items and costs is stored under the datum of every game mode, alongside the number of crystals, and the welcoming message.
-
-*/
-
 GLOBAL_LIST_EMPTY(world_uplinks)
 
-/obj/item/uplink
-	var/uses 				// Numbers of crystals
+/datum/uplink
+	var/crystals 				// Numbers of crystals
 	var/hidden_crystals = 0
 	/// List of categories with items inside
 	var/list/uplink_cats
 	/// List of all items in total (For buying random)
 	var/list/uplink_items
 
 	var/purchase_log = ""
-	var/uplink_owner = null//text-only
-	var/used_TC = 0
+	var/uplink_owner = null //text-only
```

Comment:
is this var actually used for anything?

Reply:
No, but it can be useful for admins who are investigating things.